### PR TITLE
Apr 2810 appeal successful update organisation details

### PR DESCRIPTION
**Pull request by MAli99**

### CODE REVIEW

File: src/SFA.DAS.RoatpOversight.Web/Services/ApplicationOutcomeOrchestrator.cs [hunk 1]
```diff
@@ -142,24 +140,33 @@ public async Task RecordGatewayRemovedOutcome(Guid applicationId, string userId,
...
-                        UpdatedBy = userId
+                        Username = userName,
+                        CharityNumber = registrationDetails.CharityNumber,
+                        CompanyNumber = registrationDetails.CompanyNumber,
+                        OrganisationTypeId = registrationDetails.OrganisationTypeId,
+                        ProviderTypeId = registrationDetails.ProviderTypeId,
+                        TradingName = registrationDetails.TradingName,
                     };
 
-                    await _registerApiClient.UpdateApplicationDeterminedDate(updateDeterminedDateRequest);
```
Comment (DevenKShah) [inline src/SFA.DAS.RoatpOversight.Web/Services/ApplicationOutcomeOrchestrator.cs:153]: Is the method UpdateApplicationDeterminedDate redundant now? Should we delete it?

**Review by DevenKShah — APPROVED** [review]
Looks good. Just a comment about a method that is not used which can be removed.

File: src/SFA.DAS.RoatpOversight.Web/Services/ApplicationOutcomeOrchestrator.cs (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.RoatpOversight.Web/Services/ApplicationOutcomeOrchestrator.cs:153]: agree

Comment (MarkFCain) [reply src/SFA.DAS.RoatpOversight.Web/Services/ApplicationOutcomeOrchestrator.cs:153]: I've raised a task to clean this up APR-2832, as it's already reached PO Review

**Pull request merged**